### DISCUSSION TITLE
Issue upgrading docker image from 5.0 to 6.0

### DISCUSSION THREAD

**Discussion by lombardialess** [docker-library/mongo#602] [General]
we are upgrading from a docker container with version 5.0.14 to 6.0.3 but the container keeps restarting.
We have upgraded featurecompatibilityversion from 4.0 to 5.0 before the upgrade, the log show this:

{"t":{"$date":"2023-01-11T08:33:07.890+00:00"},"s":"F",  "c":"ASSERT",   "id":23079,   "ctx":"initandlisten","msg":"Invariant failure","attr":{"expr":"isVersionInitialized()","file":"src/mongo/db/server_options.h","line":156}}
{"t":{"$date":"2023-01-11T08:33:07.890+00:00"},"s":"F",  "c":"ASSERT",   "id":23080,   "ctx":"initandlisten","msg":"\n\n***aborting after invariant() failure\n\n"}
{"t":{"$date":"2023-01-11T08:33:07.890+00:00"},"s":"F",  "c":"CONTROL",  "id":6384300, "ctx":"initandlisten","msg":"Writing fatal message","attr":{"message":"Got signal: 6 (Aborted).\n"}}

if I check src/mongo/db/server_options.h seems related to FCV but the featurecompatibility is 5.0 now, applied multiple times.
A new container with version 6.0.3 and without data starts without any issue.

**Comment by jkrains** [comment]
i meet the same issue,  please help.
